pkg/vpn_server/values.go

```diff
@@ -41,49 +41,34 @@ func BuildValues(cfg config.VPNServer) (openvpn.SeedServerValues, error) {
 		v.HAVPNClients = -1
 	}
 
-	switch cfg.IPFamilies {
-	case config.IPv4Family:
-		if len(cfg.VPNNetwork.IP) != 4 {
-			return v, fmt.Errorf("vpn network prefix is not v4 although v4 address family was specified")
-		}
-		if ones, _ := cfg.VPNNetwork.Mask.Size(); ones != 24 {
-			return v, fmt.Errorf("invalid prefixlength of vpn network prefix, must be /24, vpn network: %s", cfg.VPNNetwork)
-		}
-		vpnNetworkBytes := copyIP(cfg.VPNNetwork.IP)
-		switch v.IsHA {
-		case true:
-			vpnNetworkBytes[3] = byte(v.VPNIndex * 64)
-			v.OpenVPNNetwork = network.CIDR{
-				IP:   copyIP(vpnNetworkBytes),
-				Mask: net.CIDRMask(26, 32),
-			}
-			vpnNetworkBytes[3] = byte(v.VPNIndex*64 + 8)
-			v.IPv4PoolStartIP = vpnNetworkBytes.String()
-			vpnNetworkBytes[3] = byte(v.VPNIndex*64 + 62)
-			v.IPv4PoolEndIP = vpnNetworkBytes.String()
-		case false:
-			v.OpenVPNNetwork = cfg.VPNNetwork
-			vpnNetworkBytes[3] = byte(10)
-			v.IPv4PoolStartIP = vpnNetworkBytes.String()
-			vpnNetworkBytes[3] = byte(254)
-			v.IPv4PoolEndIP = vpnNetworkBytes.String()
-		}
+	if len(cfg.VPNNetwork.IP) != 16 {
+		return v, fmt.Errorf("vpn network prefix must be v6")
+	}
+	if ones, _ := cfg.VPNNetwork.Mask.Size(); ones != 120 {
+		return v, fmt.Errorf("invalid prefixlength of vpn network prefix, must be /120, vpn network: %s", cfg.VPNNetwork)
+	}
 
-	case config.IPv6Family:
-		if len(cfg.VPNNetwork.IP) != 16 {
-			return v, fmt.Errorf("vpn network prefix is not v6 although v6 address family was specified")
+	switch v.IsHA {
+	case false:
```

Comment:
Would an `if`/`else` block not be the more natural choice for a boolean condition?